stack.yaml

```diff
@@ -21,7 +21,7 @@ packages:
   extra-dep: true
 
 extra-deps:
-- cryptonite-0.22
+- cryptonite-0.25
```

Comment:
Is this the version they're using upstream? If this needs a specific version of above, it would be advisable to add a lower version bound to the `.cabal` file.

Reply:
done

Reply:
Wait, so does this agree with upstream cardano libraries?

Reply:
yes: https://github.com/input-output-hk/cardano-sl/blob/8e26a7ac346d84135a4e178af58c144ff80ea222/cabal.project.freeze#L44